enterprise/internal/campaigns/reconciler/executor.go

```diff
@@ -535,12 +535,26 @@ func buildCommitOpts(repo *types.Repo, extSvc *types.ExternalService, spec *camp
 	return opts, nil
 }
 
+// ErrNoSSHPush is returned by buildPushConfig if the clone URL of the
+// repository uses the ssh:// scheme, which is currently not supported by campaigns.
+type ErrNoSSHPush struct{}
+
+func (e ErrNoSSHPush) Error() string {
+	return "campaigns currently do not support pushing commits via SSH, only HTTP(s) is supported right now"
```

Comment:
Can we mention the `gitURLType` setting somehow? I wouldn't know where the SSH is coming from, since I didn't specify it in the campaign spec anywhere.
I have no great suggestion however, sorry :| 

```suggestion
	return "campaigns currently do not support pushing commits on repos with "gitURLType": "SSH", only HTTP(s) is supported right now"
```

Reply:
Yeah, good point. I changed the text to be slightly more non-site-admin welcoming, can you double-check?

Reply:
Looks good!